Dockerfile

```diff
@@ -0,0 +1,47 @@
+# Build docker container: docker build -t voxel-engine .
+# Build project: docker run --rm -it -v$(pwd):/project voxel-engine bash -c "cmake -DCMAKE_BUILD_TYPE=Release -Bbuild && cmake --build build"
+# Run project in docker: docker run --rm -it -v$(pwd):/project -v/tmp/.X11-unix:/tmp/.X11-unix -v${XAUTHORITY}:/home/user/.Xauthority:ro -eDISPLAY --network=host voxel-engine ./build/VoxelEngine
+
+FROM debian:bullseye-slim
+LABEL Description="Docker container for building VoxelEngine for Linux"
+
+# Install dependencies
+RUN apt-get update && apt-get install --no-install-recommends -y \
```

Comment:
Is there way to take this list from CMake file automatically?

Reply:
As I know there is not option to export them from CMake files. Let me know if you know how to do it, otherwise lets start with something simple and well known, then we can update/simplify it later.